### ISSUE TITLE
Direct database reference in definition

### ISSUE THREAD

**Issue by eywalker** [datajoint/datajoint-python#53]
Should we allow the table definition to contain reference to other databases directly via the database name rather than the module name? For example, should the following definition be allowed in *`Base` derivatives*

```python
definition = """
schema1.Subjects (manual)   # list of subjects
-> `database2`.Experimenter
```

where `database2` is the actual name of the database under which a table named `experimenter` actually exists. Such referencing style is currently allowed if you directly instantiate a `Table` object, passing in a definition string to the constructor.

I was thinking that since we expect all `dj.Base` derivatives to reference each other via `module.class` naming conventions, it would make sense to actually prohibit direct reference to a database from within definitions for `dj.Base` derivatives.

**Comment by dimitri-yatsenko** [comment]
i like it.

**Comment by dimitri-yatsenko** [comment]
What's the status of this?

**Comment by dimitri-yatsenko** [comment]
Fixed in 959f6b7.
